src/project-state.py

```diff
@@ -305,6 +317,10 @@ def publish_comment(text, context):
 
 
 def update_comment(ts, text, context):
+    if use_slack_api:
```

Comment:
I'd be tempted to just fail hard for the user on this and have the action fail.

Reply:
Agree, failing silently may cause confusion. Updated.